### PR TITLE
[FEAT] add method to bind to existing KVs

### PR DESCRIPTION
**Pull request by Jarema**
This method allows binding to existing KV without creating it.

### CODE REVIEW

File: nats-base-client/jsclient.ts [hunk 1]
```diff
@@ -92,6 +93,9 @@ class ViewsImpl implements Views {
   kv(name: string, opts: Partial<KvOptions> = {}): Promise<KV> {
     return Bucket.create(this.js, name, opts);
   }
+  bind(name: string, codec?: KvCodecs): Promise<KV> {
```
Comment (aricart) [inline nats-base-client/jsclient.ts:96]: this needs to be triggered by an option in KvOptions. - the views interface should simply be the raw functionality - kv, object store, etc.

**Review by aricart — CHANGES_REQUESTED** [review]

File: nats-base-client/jsclient.ts (hunk 1 above)
Comment (Jarema) [reply nats-base-client/jsclient.ts:96]: @aricart so basically additional "bind" option in KvOptions?

Comment (aricart) [reply nats-base-client/jsclient.ts:96]: yes!

Comment (Jarema) [reply nats-base-client/jsclient.ts]: fixed.

went with `bindOnly` as thats more self-explenatory and clear IMO.

File: nats-base-client/kv.ts [hunk 2]
```diff
@@ -192,6 +192,21 @@ export class Bucket implements KV, KvRemove {
     return bucket;
   }
 
+  static async bind(
+    js: JetStreamClient,
+    name: string,
+    codec: KvCodecs = NoopKvCodecs(),
```
Comment (aricart) [inline nats-base-client/kv.ts:198]: make the sig `(js: JetStreamClient, name: string, opts: {codec: KvCodec} = {})`

**Review by aricart — CHANGES_REQUESTED** [review]
LGTM - with the exception of that arg should be optional and we should prepare for other things without flipping the signature.

**Review by aricart — APPROVED** [review]
LGTM

**Pull request merged**